fastlane/Fastfile

```diff
@@ -344,12 +344,135 @@ platform :ios do
 
   desc "Build the Paywalls Test app for Emerge Snapshots"
   lane :build_paywalls_tester_for_emerge do |options|
+    build_paywalls_tester_for_emerge_ios(options)
+    build_paywalls_tester_for_emerge_mac(platform: "mac-catalyst-scaled-to-match-ipad")
+    build_paywalls_tester_for_emerge_mac(platform: "mac-catalyst-optimized-for-mac")
+
+    # As of July 2025, Emerge Tools does not support iOS apps on Mac.
+    # build_paywalls_tester_for_emerge_mac(platform: "ipad-app-on-mac")
+    # build_paywalls_tester_for_emerge_mac(platform: "iphone-app-on-mac")
```

Comment:
This comment seems to contradict the one below. If I'm understanding correctly, this will eventually be supported, right? Then maybe I'd say it like this:
```suggestion
    # As of July 2025, Emerge Tools does not support iOS apps on Mac yet.
    # build_paywalls_tester_for_emerge_mac(platform: "ipad-app-on-mac")
    # build_paywalls_tester_for_emerge_mac(platform: "iphone-app-on-mac")
```